ddsketch/pb/sketchpb/ddsketch.pb.go

```diff
@@ -4,8 +4,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.0
-// 	protoc        v3.19.4
+// 	protoc-gen-go v1.34.2
+// 	protoc        v4.25.1
```

Comment:
Why is this change? Does it come from using `gostreamer`?

Reply:
No. This is happening because this repo does not pin the version of protoc or protoc-gen-go used to generate the protobufs and these (newer versions) were what were on my system. 

The only thing that changes in this file was:
* this comment (which is a comment and thus change how the code behaves) 
* `interface{}` was rewritten to `any` which are equivalent; `any` became the recommended type to use with the introduction of generics 

https://medium.com/@csaba.ujvari/go-1-18-any-instead-of-interface-ce15dd7bea5d